jupyter-notebooks/orders_basemaps/SDK_order_basemaps.ipynb

```diff
@@ -0,0 +1,500 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Order a Basemap Using the Orders API & SDK via AOI & Quad ID"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview ##\n",
+    "---\n",
+    "In this notebook, you will learn how to order a [Planet Basemap](https://developers.planet.com/docs/data/visual-basemaps/) using your [Area of Interest](https://developers.planet.com/apis/orders/basemaps/#order-basemaps-by-area-of-interest-aoi) (AOI) and a [Quad ID](https://developers.planet.com/apis/orders/basemaps/#order-basemaps-by-quad-ids-and-deliver-to-cloud). We will place this order via Planet's [Orders API](https://developers.planet.com/apis/orders/) using our Planet Python [SDK](https://planet-sdk-for-python-v2.readthedocs.io/en/latest/python/sdk-guide/). \n",
+    "\n",
+    "1. Get a Basemap ID using either [Planet Explorer](https://developers.planet.com/docs/apps/explorer/) or the [Basemap Viewer](https://www.planet.com/basemaps/#/mosaic/). \n",
+    "2. Create a JSON order packet with order parameters.\n",
+    "3. Set up a session with the Planet SDK, and place the order, [delivering]((https://developers.planet.com/apis/orders/delivery/)) the resulting Basemap to a Google Cloud Storage bucket. \n",
+    "4. Repeat steps 1-3 using quad IDs. "
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Order a Basemap using an AOI\n",
+    "---\n",
+    "### First, find a Basemap to order\n",
+    "You can get a Basemap ID from Planet Explorer or the Basemap Viewer.\n",
+    "\n",
```

Comment:
This part seems a bit repetitive once you look at the next section. Is this supposed to be a header for this section? Consider changing the font sizes to reflect this, or just take this part out since you specify how to get a basemap ID in the next section